File: 5_circle_equation.py
import numpy as np
import matplotlib.pyplot as plt

r = 5
xc, yc = 0, 0

theta = np.linspace(0, 2*np.pi, 1000)
x = xc + r * np.cos(theta)
y = yc + r * np.sin(theta)

plt.plot(x, y)
plt.gca().set_aspect('equal', adjustable='box')
plt.grid()
plt.title("Circle Using Equation")
plt.show()

File: 5_circle_equation.m
clc; clear; close all;

r = 5;
xc = 0; yc = 0;

theta = 0:0.01:2*pi;

x = xc + r*cos(theta);
y = yc + r*sin(theta);

plot(x, y);
axis equal;
grid on;
title('Circle Using Equation');
